modules/preview-react/_examples/stories/examples/SelectWithFormik.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import {useFormik} from 'formik';
+import {Select} from '@workday/canvas-kit-preview-react/select';
+import {
+  FormField,
+  useFormFieldInput,
+  useFormFieldModel,
+} from '@workday/canvas-kit-preview-react/form-field';
+
+export const SelectWithFormik = () => {
+  const formik = useFormik({
+    initialValues: {
+      selectedBook: '',
+    },
+    onSubmit: data => {
+      console.log(data);
+    },
+  });
+
+  const model = useFormFieldModel();
+  const formFieldInputProps = useFormFieldInput(model);
+
+  return (
+    <form onSubmit={formik.handleSubmit}>
```

Comment:
Even though it is not required in HTML5, I have found  adding action placeholder, e.g.`action="."` or `action="#"`, on a form help with edge cases on mobile browsers and screen readers. In the same vain, should there be a submit button?